i18n/SETUP.md

```diff
@@ -0,0 +1,37 @@
+# Curriculum Builder I18n Sync Setup
+
+The Curriculum Builder I18n Sync runs on production daily at 7am UTC. Follow the instructions below to set up your local instance of Curriculum Builder.
+
+
+### Adding API Keys
+
+Add the following configurations to your `local_settings.py`, using the values specified in the [Heroku settings](https://dashboard.heroku.com/apps/curriculumbuilder/settings).
+
+```
+AWS_ACCESS_KEY_ID = '...'
+AWS_DEFAULT_REGION = 'us-east-1'
+AWS_SECRET_ACCESS_KEY = '...'
```

Comment:
I can't remember, do these need to be added as command line arguments as well so we can download the list of languages from DynamoDB?
For example: 
```
AWS_ACCESS_KEY_ID=... AWS_DEFAULT_REGION=us-east-1 AWS_SECRET_ACCESS_KEY... ./manage.py gather_i18n_strings
```

Reply:
oh yep, I can add that!